README.md

```diff
@@ -77,7 +77,8 @@ Mona's sdk provides a simple way to upload a new Mona configuration, using Clien
 ```
 # Note: no need to add your user_id as the key, just the configuration itself.
```

Comment:
this line should be revised now, right?

Reply:
no this is still correct:) in should_use_authentication=false mode the constructor will fail if user_id is not provided, so in both cases the client have the "user_id" field.